src/main/kotlin/graphics/scenery/Box.kt

```diff
@@ -135,6 +137,20 @@ open class Box @JvmOverloads constructor(val sizes: Vector3f = Vector3f(1.0f, 1.
         boundingBox = generateBoundingBox()
     }
 
+    override fun getConstructorParameters(): Any? {
```

Comment:
Shouldn't these two methods be static, aka, be moved to the companion object, as they are essentially factory methods?